enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,340 @@
+  - https://issues.redhat.com/browse/CCO-286
+see-also:
+  - []
+replaces:
+  - []
+superseded-by:
+  - []
+---
+
+# Tokenized Authentication for Red Hat Operators on Cloud Providers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Many Cloud Providers offer services that allow authentication via Temporary Access Tokens (TAT).
+
+For example, Amazon Web Services (AWS) Secure Token Service (STS) allows you to request temporary limited-privilege
+credentials. A concrete example is write permission on an S3 bucket on AWS. STS is supported on all the major cloud
+providers where OpenShift is deployed.
+
+Currently, several Red Hat operators provide detailed instructions for how to use the operator on clusters where CCO is
+in Manual Mode. The process is designed to have manual operations performed by the customer that are inconsistent across
+layered product operators.
+
+This enhancement seeks to bring a unified set of standards and tooling to the process of enabling an
+operator to access cloud provider resources using TAT, and improve the UX for consumers of these operators. 
+
+## Motivation
+
+Several Red Hat operators have normal operating modes where they assure their operands have required cloud 
+resources to perform properly. In Mint Mode, CCO will automatically create the IAM roles and credentials required for 
+the operator to assume the role and authenticate with the Cloud Provider.
+
+In Manual mode or more generally on clusters that support TAT (CCO not necessarily installed), an admin installing an 
+operator is required to:
+
+Extract the CredentialsRequest from the operator's image or codebase in order to know what IAM role is appropriate for 
+the operator to assume
+if using the [Cloud Credential Operator Utility](https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md), `ccoctl`:
+1. Find and download a compatible `ccoctl` binary
+2. Accept that `ccoctl` is a binary you downloaded that has god-mode access to your Cloud Provider
+3. Use `ccoctl` to create the IAM Role
+4. Use `ccoctl` to create the secret with the credentials expected by the operator in mint mode
+
+Here is a diagram for how this works on AWS:
+
+![](STS_today.jpg)
+
+This enhancement seeks to unify this process across operators so users of several of them have the same experience and
+similar steps to perform. We aim to automate and reduce those steps as much as possible to make installing operators
+(via OLM) simpler on clusters where TAT authentication is supported.
+
+### User Stories
+
+* As a cluster admin, I want to know which OLM Operators are safe to install because they will not be interacting with
```

Comment:
What does `"safe"` mean in this context?

Reply:
safe in this context means -- will work, have flipped a UX switch to indicate the operator supports TAT mode